releasetasks/templates/mobile/common_extras.yml.tmpl

```diff
@@ -14,6 +14,6 @@ build_props:
     revision: {{ revision }}
```

Comment:
This is one of the things that I'm never sure: Do we use balrog for Fennec or not? 🤣 I just kept the templates/tests in sync with desktop.